a-seed-from-nothing.sh

```diff
@@ -145,6 +161,11 @@ if ! ./dev/seed-deploy.sh; then
     ./dev/seed-deploy.sh
 fi
 
+# Run TENKS
+cd ~/kayobe
+export TENKS_CONFIG_PATH=config/src/kayobe-config/tenks.yml
+./dev/tenks-deploy-overcloud.sh ./tenks
+
```

Comment:
Do we always want to do this now, or was this just to speed up this weeks' sessions? I can see the benefit of having these deployed in advance, especially given we later have users use tenks to deploy a new hypervisor